common/Resources/ti.internal/extensions/node/buffer.js

```diff
@@ -1615,7 +1615,8 @@ const arrayIndexHandler = {
 		if (typeof propKey === 'string') {
 			const num = Number(propKey);
 			if (Number.isSafeInteger(num)) {
-				return setAdjustedIndex(target, num, value);
+				setAdjustedIndex(target, num, value);
```

Comment:
I think we should still return the value of setAdjustedIndex here, but that implementation is wrong...

Reply:
I compared against Node's behavior and setting a value on the buffer that is out of bounds is a noop.

```js 
const b = Buffer.alloc(1);
b[0] = 1;
b[1] = 2;
console.log(b);  // prints <Buffer 01>
```

That's why I removed the return value from `setAdjustedIndex` and always return `true`.